GameData/ROEngines/PartConfigs/RD180_BDB.cfg

```diff
@@ -0,0 +1,220 @@
+PART
+{
+	module = Part
+	name = ROE-RD180
+	author = CobaltWolf
+
+	category = Engine
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
```

Comment:
Can you make these parts [and your other PRs] use the tool in https://github.com/KSP-RO/ROEngines/blob/PatchingFixes/GameData/ROEngines/ROEEngineDefaults.cfg?